app/helpers/settings_helper.rb

```diff
@@ -1,2 +1,5 @@
 module SettingsHelper
+  def current_user
```

Comment:
Yup...

Left over artifact from the setting_spec exploration..

Removed..

Done

On Thu, Apr 9, 2015 at 1:14 PM, Ben Woodall <notifications@github.com>
wrote:

> In app/helpers/settings_helper.rb
> <https://github.com/benwoody/shrtnr/pull/4#discussion_r28098145>:
>
> > @@ -1,2 +1,5 @@
> >  module SettingsHelper
> > +  def current_user
>
> This is already defined in the SessionsHelper. Keep it DRY!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benwoody/shrtnr/pull/4/files#r28098145>.
>



-- 
Joe Simpson
“Reasonable people adapt themselves to the world. Unreasonable people
attempt to adapt the world to themselves. All progress, therefore, depends
on unreasonable people.”
George Bernard Shaw